resolwe_bio/processes/support_processors/rnaseqc_qc.py

```diff
@@ -102,13 +103,23 @@ def add_attributes(x):
         + df["gene_id"].astype(str)
     )
 
+    chromosome_order = df["chromosome"].unique().tolist()
```

Comment:
Is this assuming the chromosome order in the annotation is in the correct (final) order, i.e. chr1, chr2... chrX, MT? What would happen, and do we care, if chromosomes are in some other order, i.e. chr1, chr10, chr11, chr2...?